File: benchmark.c
#include <stdio.h>
#include <time.h>

int main() {
    clock_t start = clock();
    
    long long sum = 0;
    for (int i = 0; i < 10000000; i++) {
        sum += i;
    }
    
    clock_t end = clock();
    double time = (double)(end - start) / CLOCKS_PER_SEC;
    
    printf("C Loop sum = %lld\n", sum);
    printf("Time: %.3f seconds\n", time);
    
    return 0;
}

File: benchmark.js
const start = Date.now();

let sum = 0;
for (let i = 0; i < 10000000; i++) {
    sum += i;
}

const end = Date.now();

console.log(`JS Loop sum = ${sum}`);
console.log(`Time: ${(end - start) / 1000} seconds`);
